servers/zms/src/main/java/com/yahoo/athenz/zms/store/impl/jdbc/JDBCConnection.java

```diff
@@ -6713,6 +6845,31 @@ private void addTagsToGroups(Map<String, Group> groupMap, String domainName) {
         }
     }
 
+    Map<String, Map<String, TagValueList>> getDomainResourceTags(String domainName, String caller, String sqlStatement) {
+        final String funcCaller = "getDomain" + caller + "Tags";
```

Comment:
you should not edit the caller..

Reply:
it was part of the attempt to make it more generic